docs/how-tos/manage-modules.md

```diff
@@ -62,7 +62,13 @@ If you intend to make frequent code changes to your module, want to support a va
    tar -czf module.tar.gz <PATH-TO-EXECUTABLE>
```

Comment:
```suggestion
   tar -czf module.tar.gz <PATH-TO-EXECUTABLE> <OTHER-REQUIRED-FILES-AS-NEEDED>
```

Reply:
I like the idea of using tabs better to provide more clarity

Reply:
use tabs